src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,56 @@
+package calculator;
+
+public class Calculator {
+
+    public static final String DELIMITER = ",|:";
+
+    public static int cal(String text) {
+        String customDelimiter = "";
+
+        if (isBlank(text)) {
+            return 0;
+        }
+        if (text.startsWith("//")){
+            String[] result = split(text, "//|\n");
+            customDelimiter = result[1];
+            text = result[2];
+
+            return sum(toInts(split(text, customDelimiter)));
+        }
```

Comment:
```suggestion
        if (text.startsWith("//")){
            String[] result = split(text, "//|\n");
            customDelimiter = result[1];
            text = result[2];

            return sum(toInts(split(text, customDelimiter)));
        }
```

해당 로직을 이해하거나 테스트코드 혹은 전체 로직을 이해해야지 `"//"`,`"//|\n"` 에 대한 의미를 이해할 수 있을것 같아요!
사람이 이해하기 쉽게 의미있는 이름의 변수로 선언해보는건 어떨까요!?

```
public class Calculator {

    public static final String DELIMITER = ",|:";
    public static final String CUSTOM_START_TEXT = "//" ;
```

Reply:
![스크린샷 2023-11-05 오후 8 09 31](https://github.com/next-step/java-racingcar/assets/12059592/9819c5be-0092-441f-946b-83a945ad3457)

변하는것과 변하지 않는것을 잘 고민해보면,
`sum(toInts(split(text, {변하는부분})));`
반복되는 패턴이 보입니다!

그렇다면, 구분자를 추출하는 책임을 가진 메소드가 있으면 로직이 훨씬 간단해지지 않을까요!?